github/deploy.py

```diff
@@ -41,7 +41,7 @@ def parse_deployment_properties(fn):
     return deployment_properties
 
 
-targets = "{targets}".split(',')
+targets = list(filter(None, "{targets}".split(',')))
```

Comment:
I am proposing this one as I think it's less terse (especially considering that most of us here aren't Python programmers:

```
targets = [] if "{targets}".split(',') == [''] else "{targets}".split(',')
```

Reply:
@lolski we can simplify it as:
`targets = [] if "{targets}" else "{targets}".split(',')`

Reply:
that is even better @vmax , well done